modules/navier_stokes/examples/flow_over_sphere_fv.i

```diff
@@ -0,0 +1,208 @@
+    two_term_boundary_expansion = false
+  []
+  [vel_z]
+    type = INSFVVelocityVariable
+    two_term_boundary_expansion = false
+  []
+  [pressure]
+    type = INSFVPressureVariable
+    two_term_boundary_expansion = false
+  []
+[]
+
+[FVKernels]
+  # mass
+  [mass]
+    type = INSFVMassAdvection
+    variable = pressure
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    rho = ${rho}
+  []
+
+  [u_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_x
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    rho = ${rho}
+    momentum_component = 'x'
+  []
+  [u_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_x
+    mu = ${mu}
+    momentum_component = 'x'
+  []
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_x
+    momentum_component = 'x'
+    pressure = pressure
+  []
+
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_y
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_y
+    mu = ${mu}
+    momentum_component = 'y'
+  []
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_y
+    momentum_component = 'y'
+    pressure = pressure
+  []
+
+  [w_advection]
+    type = INSFVMomentumAdvection
+    variable = vel_z
+    advected_interp_method = ${advected_interp_method}
+    velocity_interp_method = ${velocity_interp_method}
+    rho = ${rho}
+    momentum_component = 'z'
+  []
+  [w_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = vel_z
+    mu = ${mu}
+    momentum_component = 'z'
+  []
+  [w_pressure]
+    type = INSFVMomentumPressure
+    variable = vel_z
+    momentum_component = 'z'
+    pressure = pressure
+  []
+[]
+
+[FVBCs]
+  [no_slip_x]
+    type = INSFVNoSlipWallBC
+    variable = vel_x
+    boundary = 'no_slip inlet'
+    function = 0
+  []
+
+  [no_slip_y]
+    type = INSFVNoSlipWallBC
+    variable = vel_y
+    boundary = 'no_slip inlet'
+    function = 0
+  []
+
+  [no_slip_z]
+    type = INSFVNoSlipWallBC
+    variable = vel_z
+    boundary = 'no_slip'
+    function = 0
+  []
+
+  [inlet_z]
+    type = INSFVInletVelocityBC
+    variable = vel_z
+    boundary = 'inlet'
+    function = inlet_func
+  []
+
+  [outlet_p]
+    type = INSFVOutletPressureBC
+    variable = pressure
+    boundary = 'outlet'
+    function = 0
+  []
+[]
+
+[Functions]
+  [./inlet_func]
+    type = ParsedFunction
+    value = 'sqrt((x-2)^2 * (x+2)^2 * (y-2)^2 * (y+2)^2) / 16'
+  [../]
+[]
+
+[Materials]
+  [./const]
+    type = GenericConstantMaterial
+    prop_names = 'rho mu'
+    prop_values = '${rho}  ${mu}'
+  [../]
+[]
+
+[Preconditioning]
+  [./SMP] #What is PJFNK
+    type = SMP
+    full = true
+    solve_type = 'NEWTON'
+  [../]
```

Comment:
```suggestion
  [SMP]
    type = SMP
    full = true
    solve_type = 'NEWTON'
  []
```

Reply:
documentation for PJFNK is here
https://mooseframework.inl.gov/source/executioners/Steady.html

Reply:
A NEWTON solve type automatically sets a full preconditioner, so putting `solve_type = NEWTON` in the `Executioner` allows removal of the `Preconditioning` block entirely